wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -694,7 +695,13 @@ def _add_to_launch_queue(self, run: SweepRun) -> bool:
             )
 
         # override resource and args of job
-        _job_launch_config = self._wandb_run.config.get("launch") or {}
+        _job_launch_config = copy.deepcopy(self._wandb_run.config.get("launch")) or {}
+
+        # if we have both resource args and template vars, only pass template vars
+        resource_args = _job_launch_config.get("resource_args")
+        template_variables = _job_launch_config.get("template_variables")
+        if resource_args and template_variables:
+            resource_args = None
```

Comment:
Can we log here, we should prevent the resource_args from being sent to the scheduler if template vars are given, so logging could indicate to us if something is going wrong

Reply:
done